### PR TITLE
added: auth

### PR DESCRIPTION
**Pull request by omgaikwad**
Added auth:

auth-context
user signup
user login

### CODE REVIEW

File: src/pages/Login/Login.jsx [hunk 1]
```diff
@@ -1,14 +1,90 @@
...
+        isLoggedIn: true,
+        token: response.data.encodedToken,
+        user: {
+          firstName: response.data.foundUser.firstName,
+          lastName: response.data.foundUser.lastName,
+          email: response.data.foundUser.email,
+        },
+      });
+      console.log(response);
+
```
Comment (prathmeshjagtap) [inline src/pages/Login/Login.jsx]: you can remove console.log

File: src/pages/Signup/Signup.jsx [hunk 2]
```diff
@@ -1,91 +1,277 @@
...
+            token: response.data.encodedToken,
+            user: {
+              firstName: response.data.createdUser.firstName,
+              lastName: response.data.createdUser.lastName,
+              email: response.data.createdUser.email,
+            },
+          });
+
+          console.log(response);
+
```
Comment (prathmeshjagtap) [inline src/pages/Signup/Signup.jsx]: you can remvoe console.log

**Review by prathmeshjagtap — APPROVED** [review]
Liked the code,it is clean and well structured
UI is Also good

**Review by t007rushi — APPROVED** [review]
Looks Good to merge

**Pull request merged**